### ISSUE TITLE
Remove 7dav cumulative signals from JHU documentation and from the DB

### ISSUE THREAD

**Issue by nmdefries** [cmu-delphi/delphi-epidata#1527]
In JHU Cases and Deaths, the documentation still lists the silly signals `confirmed_7dav_cumulative_num` and `confirmed_7dav_cumulative_prop` -- it doesn’t make much sense to average cumulative data.  We should remove these from the documentation, and also from the DB.  Of note, the parallel source USAFacts does not have these (compare the two signal lists).  I [Roni] have a vague recollection that I pointed this out during the pandemic, and it was removed, but maybe only from USAFacts and not from JHU.

**Comment by carlynvandyke** [comment]
@RoniRos just want to confirm- we should remove `deaths_7dav_cumulative_num` and `deaths_7dav_cumulative_prop`, too, right?

**Comment by RoniRos** [comment]
Yes!  And if you find the combination of "7dav" and "cumulative" anywhere else, that should probably be removed as well!

**Comment by melange396** [comment]
here are all of the cumulative and averaged signals i can find from the metadata:
```
    ('indicator-combination', 'confirmed_7dav_cumulative_num'),
    ('indicator-combination', 'confirmed_7dav_cumulative_prop'),
    ('indicator-combination', 'deaths_7dav_cumulative_num'),
    ('indicator-combination', 'deaths_7dav_cumulative_prop'),
    ('jhu-csse', 'confirmed_7dav_cumulative_num'),
    ('jhu-csse', 'confirmed_7dav_cumulative_prop'),
    ('jhu-csse', 'deaths_7dav_cumulative_num'),
    ('jhu-csse', 'deaths_7dav_cumulative_prop'),
    ('usa-facts', 'confirmed_7dav_cumulative_num'),
    ('usa-facts', 'confirmed_7dav_cumulative_prop'),
    ('usa-facts', 'deaths_7dav_cumulative_num'),
    ('usa-facts', 'deaths_7dav_cumulative_prop')
```
they still need to be removed from the database, so i am going to edit the associated PR to remove the "automatic close" linkage
